pages/faq/questions.md

```diff
@@ -132,13 +132,13 @@ BalenaOS comes in two flavors, a rolling release and an [Extended Support Releas
 
 ##### When are device types discontinued?
 
-Device types are [discontinued](#what-does-it-mean-when-a-device-type-is-discontinued) in {{ $names.os.lower }} when the manufacturer discontinues support for them. This usually happens either when the device is end-of-life and manufacturing is stopped, or when software updates by the manufacturer stop happening and updating the device type's software is no longer possible. Customers using discontinued devices do it at their own risk and are advised to move to a different platform as soon as possible.
+Device types are [discontinued](#what-does-it-mean-when-a-device-type-is-discontinued) in {{ $names.os.lower }} based on criteria outlined [here](/pages/reference/hardware/versioning.md#discontinued)
```

Comment:
I've tried to de-duplicate here, so we just maintain a single section, and point to it from the faqs

Reply:
good shout!